packages/web-components/fast-components-msft/src/progress/progress/progress.styles.ts

```diff
@@ -2,7 +2,7 @@ import { css } from "@microsoft/fast-element";
 import { SystemColors } from "@microsoft/fast-web-utilities";
 import { display, forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
 import {
```

Comment:
Do we *also* want to make this change in `@microsoft/fast-components`?

Reply:
Yes, good point. Fixed.